### ISSUE TITLE
when i train LISA_DATA following your instruction, i can not detect traffic-sign(stop, pedestrianCrossing)

### ISSUE THREAD

**Issue by gjghks** [georgesung/ssd_tensorflow_traffic_sign_detection#5]
first, thank you for your study.

i train data following your instruction.

i have 2,600 resized_images_400x260 using create_pickle.py.

before that, 6,618 annotations was acquired by using mergeAnnotationFiles.py and extractAnnotations.py.

then use data_prep.py and train.py

this is train result.
----------------------------------------------------------------------------------------------------------------
Epoch 196 -- Train loss: 18.6558, Validation loss: 49.2677, Elapsed time: 11.47 sec
Epoch 197 -- Train loss: 18.7084, Validation loss: 49.3180, Elapsed time: 11.46 sec
Epoch 198 -- Train loss: 18.5914, Validation loss: 47.7009, Elapsed time: 11.41 sec
Epoch 199 -- Train loss: 18.5991, Validation loss: 53.4686, Elapsed time: 11.43 sec
Epoch 200 -- Train loss: 19.2694, Validation loss: 50.9338, Elapsed time: 11.43 sec
Total elapsed time: 38 min 10 sec
Trained model saved at: ./model.ckpt
Loss history saved at loss_history.p
---------------------------------------------------------------------------------------------------------------

train loss and validation have high rate.

so, i can not detect any traffic-signs in demo.

i do not change any setting..

hope for your help.

**Comment by Suodislie** [comment]
@gjghks Do you have solved this problem?  i meet the same issue with you, but my loss is close to 2, i change the bitch_size to 2

**Comment by georgesung** [comment]
Can you try resuming learning with a lower learning rate? How does your loss curve look like?

**Comment by alonebaby** [comment]
I have the same issue, the train loss is 7.6, validation loss is 7.9, after 6000 epoch, the learning rate is 0.01, batch size 32
